index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Nandita Gilroy's Personal Portfolio</title>
+        <meta name="description" content="Nandita Gilroy's Personal Portfolio Site">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="styles/stylesheet.css">
+    </head>
+    <body>
+        <nav id="nav-background">
+            <ul>
+                <li><a href="index.html">Home</a> </li>
+                <li><a href="portfolio.html">Portfolio</a> </li>
+                <li><a href="hobby-blog.html">Hobbies</a> </li>
+                <li><a href="about.html">About Me</a> </li>
+            </ul> 
+        </nav>
+        <header>
+            <h1>Nandita Gilroy</h1>
+            <h3>I do code.</h3>
```

Comment:
Just be aware that the white text here, against the now capped peaks can be harder to read.